package.json

```diff
@@ -16,7 +16,8 @@
     "typecheck": "tsc --noEmit",
     "build:esm": "tsc",
     "build:cjs": "tsc --module commonjs --outDir dist/cjs",
-    "build": "rm -rf dist && rm -f .tsbuildinfo && npm run build:esm && npm run build:cjs",
+    "build:type-augmentation": "cp -r src/type-augmentation dist/type-augmentation",
```

Comment:
This exists because Typescript does not by default include any source `.d.ts` files in the output.

Potentially these could just be `.ts` files, in whcih case they would be included. But keeping them as `.d.ts` 100% ensures they have no runtime impact.